feature-libs/order/components/order-details/order-detail-reorder/reorder-dialog/reorder-dialog.component.html

```diff
@@ -0,0 +1,81 @@
+<div
+  [cxFocus]="focusConfig"
+  (esc)="close('Escape clicked')"
+  class="cx-modal-container"
+>
+  <div class="cx-modal-content">
+    <ng-container>
+      <div class="cx-dialog-header modal-header">
+        <div class="cx-dialog-title modal-title">
+          {{ 'reorder.dialog.reorderProducts' | cxTranslate }}
+        </div>
+        <button
+          type="button"
+          class="close"
+          attr.aria-label="{{ 'addToCart.closeModal' | cxTranslate }}"
+          (click)="close('Close Import Products Dialog')"
+        >
+          <span aria-hidden="true">
+            <cx-icon [type]="iconTypes.CLOSE"></cx-icon>
+          </span>
+        </button>
+      </div>
+    </ng-container>
+
+    <!-- Modal Body -->
+    <div class="cx-dialog-body modal-body">
+      <ng-container *ngIf="!(loading$ | async); else loading">
+        <p>
+          <ng-container *ngIf="!cartModifications.length; else cartResponse">
+            <span class="cx-cart-mod-entry-container">
+              <span class="alert-icon">
+                <cx-icon class="success" [type]="iconTypes.SUCCESS"></cx-icon>
+              </span>
+              {{ 'reorder.dialog.messages.success' | cxTranslate }}
+            </span>
+          </ng-container>
+          <ng-template #cartResponse>
```

Comment:
What can I do to trigger the low stock and other error use cases (even by forcing them in the code somehow) in order to assess the behaviour of the modal?